### PR TITLE
polylith: Update to version 0.2.19

### PR DESCRIPTION
**Pull request by olymk2**
**Summary**

Update to latest release
Changelog available [here](https://github.com/polyfy/polylith/releases/tag/v0.2.19)

**Test Plan**

Built and installed locally, ran poly version to check new version is returned.

**Checklist**

- [X] Package was built and tested against unstable

### CODE REVIEW

File: packages/p/polylith/pspec_x86_64.xml [hunk 1]
```diff
@@ -9,14 +9,14 @@
         <License>EPL-1.0</License>
         <PartOf>programming.tools</PartOf>
         <Summary xml:lang="en">Clojure tool for managing polylith projects</Summary>
-        <Description xml:lang="en">The poly tool helps you manage your polylith projects which introduces the architectural concept of “service level building blocks” which can be combined like LEGO® bricks to build our services and systems.
+        <Description xml:lang="en">The poly tool helps you manage your polylith projects which introduces the architectural concept of &#x201c;service level building blocks&#x201d; which can be combined like LEGO&#xae; bricks to build our services and systems.
```
Comment (davidjharder) [inline packages/p/polylith/pspec_x86_64.xml:12]: Some sort of encoding error here. Please fix

**Review by davidjharder — CHANGES_REQUESTED** [review]

File: packages/p/polylith/pspec_x86_64.xml (hunk 1 above)
Comment (davidjharder) [reply packages/p/polylith/pspec_x86_64.xml]: Hmmm, this might be a solus tooling bug then.

Comment (davidjharder) [reply packages/p/polylith/pspec_x86_64.xml]: Yeah this is a bug, I get the same. To get around this, please replace the non-ascii quotes with ascii quotes and delete the (registered) character

Comment (olymk2) [reply packages/p/polylith/pspec_x86_64.xml]: @davidjharder quotes replaced in spec and package file.

**Comment by davidjharder** [comment]
Please squash your commits. The resulting commit should have the title and description from the first one here: 
Capture

**Comment by olymk2** [comment]
@davidjharder squashed, I am not even sure how those came to be not commits messages I made so I don't 100% get why they existed

**Pull request merged**